release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -68,6 +68,22 @@ You can now remove or rotate the GCP admin-level credential that the xref:../ope
 [id="ocp-4-7-storage"]
 === Storage
 
+[id="ocp-4-5-persistent-storage-csi-ebs"]
+==== Persistent storage using the GCP PD CSI Driver Operator (Technology Preview)
+
+The Google Cloud Platform (GCP) persistent disk (PD) CSI driver is automatically deployed and managed on GCP environments, allowing you to dynamically provision these volumes without having to install the driver manually. The GCP PD CSI Driver Operator that manages this driver is in Technology Preview.
+
+[id="ocp-4-7-storage-vsphere-problem-detector"]
+==== vSphere Problem Detector Operator
+
+The vSphere Problem Detector Operator periodically checks functionality of {product-title} clusters installed in a vSphere environment. The vSphere Problem Detector Operator is installed by default by the Cluster Storage Operator, allowing you to quickly identify and troubleshoot common storage issues, such as configuration and migration, on vSphere clusters.
+
```

Comment:
In OCP 4.7, I didn't find the migration detection is supported. Looks like only some permission and configuration checks are supported.

Reply:
Thanks, @qinpingli - I've replaced "configuration and migration" with "configuration and permissions"